### PR TITLE
MTM-58098 release notes for Date-Filtering-doesnt-work-in-Audit-Logs-…

### PR DESCRIPTION
**Pull request by pkepa7**
…for-Alarms

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+date: 2024-04-26
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md:3]:
```suggestion
date: 
```

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md:3]: The date refers to the deployment date on Zone 1 and will be added at a later time

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact: 
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58098
+version: 10.20.419.0
+---
+
+To create an audit for a repeating alarm, the alarm creation date was used, which resulted in the audit logs having past dates.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md:19]:
```suggestion
To create audit log for repeating alarms, the alarm creation date was used, which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.
```

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+
+date: 2024-04-26
+title: Create an audit log with the date of the last alarm update
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md:4]:
```suggestion
title: Audit logs for repeating alarms are created with the date of the last alarm update
```

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md (hunk 2 above)
Comment (pkepa7) [reply content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md]: Changed.

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md (hunk 3 above)
Comment (pkepa7) [reply content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md]: Changed.

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md (hunk 1 above)
Comment (pkepa7) [reply content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md]: Changed.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact: 
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-58098
+version: 10.20.419.0
+---
+
+To create audit log for repeating alarms, the alarm creation date was used, which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.
```
Comment (pink-mongoose) [inline content/change-logs/platform-services/cumulocity-10.20.419.0_Date-Filtering-doesnt-work-in-Audit-Logs-for-Alarms.md:19]:
```suggestion
Audit logs for repeating alarms used the alarm creation date which resulted in audit logs having past dates. This issue has been fixed and now audit logs for repeating alarms are created with the date of the last alarm update.
```

**Review by pink-mongoose — CHANGES_REQUESTED** [review]

**Comment by BeateRixen** [comment]
@martin-sag This ones needs approval from you please.

**Review by martin-sag — APPROVED** [review]

**Pull request merged**